src/saveload/saveload.h

```diff
@@ -304,6 +304,8 @@ enum SaveLoadVersion : uint16 {
 	SLV_ENDING_YEAR,                        ///< 218  PR#7747 v1.10 Configurable ending year.
 	SLV_REMOVE_TOWN_CARGO_CACHE,            ///< 219  PR#8258 Remove town cargo acceptance and production caches.
 
```

Comment:
I think this newline shouldn't be here :)

Reply:
The versions are grouped by five, and this is the first piece in the next group. So I think it should be. :)

Reply:
Valid point! Did not see that :D Cheers!